webpack/assets/javascripts/react_app/components/Bookmarks/BookmarksConstants.js

```diff
@@ -0,0 +1,6 @@
+export const BOOKMARKS_REQUEST = 'BOOKMARKS_REQUEST';
+export const BOOKMARKS_SUCCESS = 'BOOKMARKS_SUCCESS';
+export const BOOKMARKS_FAILURE = 'BOOKMARKS_FAILURE';
+export const BOOKMARKS_MODAL_OPENED = 'BOOKMARKS_MODAL_OPENED';
+export const BOOKMARKS_MODAL_CLOSED = 'BOOKMARKS_MODAL_CLOSED';
+export const BOOKMARK_FORM_SUBMITTED = 'BOOKMARK_FORM_SUBMITTED';
```

Comment:
Why only this one singular and the rest plural?  Would it be better to call it `BOOKMARKS_FORM_SUBMITTED`?

Reply:
Mainly because it is a form for one bookmark and also what [submitForm](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/redux/actions/common/forms.js#L45) does with `item`, but I made changes to have it in plural.